internal/worker/worker.go

```diff
@@ -51,3 +58,19 @@ func createBucket(m *mosaic.Mosaic, cfg *config.Config, stg storage.Storage) err
 
 	return stg.CreateBucket(cfg.S3.BucketName)
 }
+
+func keepAlive(ctx context.Context, lock locking.Lock) {
+	ticker := time.NewTicker(KeepAliveInterval)
+
+	for {
+		select {
+		case <-ctx.Done():
+			ticker.Stop()
+			return
+		case <-ticker.C:
+			if err := lock.Refresh(ctx, LockingTimeTTL); err != nil {
+				log.Println("failed to refresh lock TTL, error=%w", err)
+			}
```

Comment:
A good approach here is send the errors to a channel. What do you think?

Also, I think we need to use the logger was injected through uber fx.

Reply:
@mauricioabreu,

It's good idea, although, in error case, we should to stop ffmpeg process? 

Whether yes, we need change `exec.Command` to `exec.CommandContext` and other smalls things to works correctly.

What do think? Any other suggestion about?

Reply:
We don't need to stop any ffmpeg process. What I was actually trying to say is, instead of logging inside the internal function, we can log errors one level above. I don't think it makes a big difference. If you feel it is not important, you can leave it the way it is right now.

Reply:
Oh right, i did understand another thing haha.

No problem, we can try this way, see this example, would it be something like this?

![code-example](https://github.com/mauricioabreu/mosaic-video/assets/34068867/dfa18ef7-8d4e-4a94-9f8e-27369fc66f0b)

Reply:
Good but I think the code got worse. I mean, it is technically better but maybe Golang makes it unreadable for some reason. You can keep the way it was, much more readable. If anything goes wrong (or there are more errors to handle) we can use the approach in the image.

Reply:
Yes, how to get an error from channel it's a blocking code, we need to use a go routine to catch the error.

For now, using only logger to print error is enough.